.circleci/config.yml

```diff
@@ -30,13 +30,15 @@ jobs:
             aws s3 cp s3://com-stitchdata-dev-deployment-assets/environments/tap-tester/sandbox dev_env.sh
             source dev_env.sh
             source /usr/local/share/virtualenvs/tap-tester/bin/activate
-            run-a-test --tap=tap-stripe \
+            pip install 'stripe==2.42.0'
```

Comment:
This should be `pip install .[test]`, if the dependency is going to be managed in the `extras_require`

Reply:
This line is necessary because `pip install .[test]` errors and fails to install `2.42.0`

Reply:
Then I think the `test` dependencies should be removed if they're not used.

It's a bit weird in that case that this only exists in the circle file, but I think having a block that's not doing anything is a form of misdirection that doesn't sit well with me.